src/test/java/org/folio/circulation/domain/ItemTest.java

```diff
@@ -0,0 +1,50 @@
+package org.folio.circulation.domain;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.nullValue;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import java.util.UUID;
+
+import org.junit.Test;
+
+import api.support.builders.HoldingBuilder;
+import api.support.builders.ItemBuilder;
+import lombok.val;
+
+public class ItemTest {
```

Comment:
Could it worth making this class specific to location tests e.g. `ItemLocationTests` to keep the scope of the tests small?

Reply:
Renamed to `ItemPermanentLocationTest`